template/{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/databricks.yml.tmpl

```diff
@@ -24,6 +24,7 @@ targets:
     workspace:
       # TODO: add dev workspace URL
       host:
+{{ if (eq .input_setup_cicd_and_project `CICD_and_Project`)}}
```

Comment:
Why are we removing the test, staging and prod? 
After setting up CICD, users will have to add the block themselves.

It's better to add a dummy host name & comments than removing the block.
The token can be overwritten in the env var. I think the host can be overwritten too.
In this case users won't need additional step after running the cicd deployment workflow.

Reply:
Actually in the deploy CI/CD workflow, we modify `databricks.yml` so they don't have to add it in themselves

Reply:
Oh I see. It depends on the assumption that users won't modify the `databricks.yml` and make it incompatible. For example, if user removes the empty lines or add another element after `targets`, the logic will be broken.
Let's also create a ticket to follow up and not modify the project `databricks.yml`.

Reply:
Sometime in the future, we may be able to use LLM to analysis the `databricks.yml` file and safely make the updates. Until then, the best way is to not modify it, or have a reserved placeholder block for auto generated code.

Reply:
Follow-up or just update the repo now to have dummy URLs instead?

Reply:
It's fine for this PR.
Adding the `host` to env var will need a manual test to make sure it works as I'm not 100% confident.
Having a working version is more important as it's already huge.

Reply:
Filed https://databricks.atlassian.net/browse/MLOPS-613